code/game/objects/structures/crates_lockers/closets/secure/craftable/standard.dm

```diff
@@ -0,0 +1,167 @@
+#define PERSONAL 1
+#define DEPARTMENTAL 2
+#define FREE_ACCESS 3
+
+///Secure closets that can be hand crafted and have their access requirments modified
+/obj/structure/closet/secure_closet/custom
+	desc = "Secured closed with modifiable access"
```

Comment:
```suggestion
	desc = "Secure closet"
```
it's probably safe to drop the last part since we only ever have closets and secure closets